### DISCUSSION TITLE
Tasty Target causes passive sonar to be able to see as far as active sonar without alerting enemies.

### DISCUSSION THREAD

**Discussion by BlackMoons** [FakeFishGames/Barotrauma#13671] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When an assistant gets tasty target, a huge noise appears on passive sonar where they are that lights up the entire sonar range with sonar reflections. 

Enemies don't seem to notice this noise either, allowing you to see enemies on passive sonar that can't see you at all. Makes the game a LOT easier as passive sonar works as well as active. 

Expected behavior: To not be able to see the level in passive sonar unless something is actually making a noise that attracts enemies. 

### Reproduction steps

Start campaign
Spawn assistant, Observe minimal noise on sonar in passive mode. 

give assistant enough EXP to reach tasty target and take the trait

Notice passive sonar mode showing a HUGE noise where the assistant is, as loud as a large engine at full throttle. Can see the level via noise reflection almost as well as active sonar, even with the ships engines/reactor/etc off. 

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
xd

And yes, you can do that with regular sonar beacon too, but somehow, clown beacon is less annoying

**Reply by BlackMoons** [reply]
Sonar beacon you can at least turn off, and will run outta batteries.

I can't turn the stupid assistant off short of editing my savegame -_-; And its ruining all the surprise of enemies being where I can't see them on sonar till I get close enough for them to attack.

**Comment by Regalis11** [comment]
Thank you for the report! [I created an issue ticket](https://github.com/FakeFishGames/Barotrauma/issues/13676) about this in the issue tracker.
